TEST_APPS/testcases/example/test_cmdline.py

```diff
@@ -0,0 +1,50 @@
+"""
+Copyright 2018 ARM Limited
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+from icetea_lib.bench import Bench
+
+
+class Testcase(Bench):
+    def __init__(self):
+        self.device = None
+        Bench.__init__(self,
+                       name="test_cmdline",
+                       title="Smoke test for command line interface",
+                       status="released",
+                       purpose="Verify Command Line Interface",
+                       component=["cmdline"],
+                       type="smoke",
+                       requirements={
+                           "duts": {
+                               '*': {
+                                   "count": 1,
+                                   "type": "hardware",
+                                   "application": {
+                                       "name": "TEST_APPS-device-exampleapp"
+                                   }
+                               }
+                           }
+                       }
+                       )
+
+    def setup(self):
+        self.device = self.get_node_endpoint(1)
```

Comment:
this could be just `pass`, like in teardown (row 47 can be then also remove) .. Just simplify example...

Reply:
I think that this is simple enough now. It is easier if core functionality is visible in example.

Reply:
I don’t think that get_node_endpoint() is core function that should be available in this example - its more for mesh node related example...